src/Benchotron/UI/Console.purs

```diff
@@ -94,9 +95,10 @@ runBenchmarkConsole benchmark = do
   lift $ do
     stderrWrite "\n"
     noteTime \t -> "Finished at: " <> t <> "\n"
-  return r
+  pure r
   where
-  noteTime f = now >>= toLocaleTimeString >>= (stderrWrite <<< f)
+  noteTime f = nowString >>= (stderrWrite <<< f)
+  nowString = JSD.toTimeString <$> JSD.fromDateTime <$> DDI.toDateTime <$> now
```

Comment:
Perhaps this is a bit minor, but I think it's a bit easier to read using function composition `<<<` instead, so that map `<$>` only needs to appear once. Would something like `(JSD.toTimeString <<< JSD.fromDateTime <<< DDI.toDateTime) <$> now` work?